firefox-ios/Client/Frontend/Browser/MainMenu/Redux/MainMenuState.swift

```diff
@@ -134,4 +155,34 @@ struct MainMenuState: ScreenState, Equatable {
             )
         }
     }
+
+    private func getAccountData() -> AccountData? {
+        let rustAccount = RustFirefoxAccounts.shared
+        let needsReAuth = rustAccount.accountNeedsReauth()
+
+        guard let userProfile = rustAccount.userProfile else {
+            return nil
+        }
+
+        let title: String = {
+            if needsReAuth {
+                return .MainMenu.Account.SyncErrorTitle
+            }
+            return userProfile.displayName ?? userProfile.email
+        }()
```

Comment:
```suggestion
        let title: String = if needsReAuth {
                return .MainMenu.Account.SyncErrorTitle
            } else {
                return userProfile.displayName ?? userProfile.email
            }
```
An alternative; but if you use this, please indent with Xcode because I just copy pasted here. No strong opinions on which to use, just pointing out the option.

Reply:
Just got some errors after changed the code, maybe something else should be changed. Lets keep this, as it is.